proposals/0258-property-wrappers.md

```diff
@@ -469,13 +469,13 @@ struct Atomic<Value> {
   }
   
   func load(order: MemoryOrder = .relaxed) { ... }
-  mutating func store(newValue: Value, order: MemoryOrder = .relaxed) { ... }
-  mutating func increment() { ... }
-  mutating func decrement() { ... }
+  func store(newValue: Value, order: MemoryOrder = .relaxed) { ... }
+  func increment() { ... }
+  func decrement() { ... }
```

Comment:
These two don’t make much sense to me — `Value` isn’t constrained to a protocol that supports these.